### DISCUSSION TITLE
Possible to modify code to "remove" either the inputs or outputs ?

### DISCUSSION THREAD

**Discussion by jbuecker** [ExistentialAudio/BlackHole#368] [Feature Requests]
I have successfully complied a separate version to run two concurrent versions Of Blackhole.  Would it be hard to "Neuter" the outputs or the inputs in the corresponding versions?  Thanks in advance!
~JB

**Comment by devinroth** [comment]
What do you mean by neuter?

Devin Roth

**Reply by jbuecker** [reply]
Hi ,
What I meant by 'neuter' is to disable the outputs or inputs without really removing the code.... 

Was a poor choice of wording ... I would like to have a version that simply has 16 inputs without any outputs and vice versa ... 16 outputs with no inputs.

Thanks,
~JB

**Comment by devinroth** [comment]
Right. Didn’t we discuss this a while back?

Yeah it’s possible but at that point you’re going to need to change quite a bit of code. Then you need to have the two drivers communicate with each other.

What you’ll want to do is have it all in the same driver with two devices. One for input and the other for output.

It’s a little complicated.

Devin Roth
